.stickler.yml

```diff
@@ -1,7 +1,7 @@
 ---
 linters:
   phpcs:
-    standard: CakePHP
+    standard: CakePHP3
```

Comment:
It's called `CakePHP3` now? Don't remember such a change being done for core.

Reply:
Well that's exactly where I stole it from:

https://github.com/cakephp/cakephp/commit/8a10d4e81ed190d8129e0c1be07dfb93b2732a78